src/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ux-aspects",
+  "version": "1.1.0-beta.1",
+  "description": "Open source user interface framework for building modern, responsive, mobile big data applications",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UXAspects/UXAspects.git"
+  },
+  "keywords": [
+    "ux",
+    "aspects",
+    "ui",
+    "framework",
+    "angular",
+    "angularjs",
+    "bootstrap"
+  ],
+  "author": "Ashley Hunter",
+  "license": "Apache-2.0",
```

Comment:
I suppose we would have to ask Andy, but my guess is that we are using the most permissive license (MIT) due to the fact that we currently bundle so much open source code with various licenses.

The MIT license appears here, although slightly modified:
https://uxaspects.github.io/UXAspects/#/licenses